src/plat/tqma8xqp1gb/config.cmake

```
@@ -0,0 +1,35 @@
+#
+# Copyright 2021, Breakaway Consulting Pty. Ltd.
+#
+# SPDX-License-Identifier: GPL-2.0-only
+#
+
+cmake_minimum_required(VERSION 3.7.2)
+
+declare_platform(tqma8xqp1gb KernelPlatformTqma8xqp1gb PLAT_TQMA8XQP1GB KernelArchARM)
+
+if(KernelPlatformTqma8xqp1gb)
+    declare_seL4_arch(aarch64)
+    set(KernelArmCortexA35 ON)
+    set(KernelArchArmV8a ON)
+    config_set(KernelARMPlatform ARM_PLAT ${KernelPlatform})
+    set(KernelArmVtimerUpdateVOffset OFF)
+    set(KernelArmDisableWFIWFETraps ON)
+    list(APPEND KernelDTSList "tools/dts/${KernelPlatform}.dts")
+    list(APPEND KernelDTSList "src/plat/tqma8xqp1gb/overlay-${KernelPlatform}.dts")
+    declare_default_headers(
+        TIMER_FREQUENCY 8000000llu
```

This doesn't compile as the `llu` literal gets doubled up by some C header now.
```suggestion
        TIMER_FREQUENCY 8000000
```

---

Fixed.